src/Runtime.Base/src/System/Runtime/ExceptionHandling.cs

```diff
@@ -92,7 +92,7 @@ internal static void BeginSecondPass()
         }
     }
 
-    internal static unsafe class EH
+    internal static unsafe partial class EH
```

Comment:
I was going to move these additions to a ExceptionHandling.wasm.cs file, would that be better?

Reply:
Yep